sweep/walletsweep.go

```diff
@@ -71,14 +71,26 @@ func DetermineFeePerKw(feeEstimator chainfee.Estimator,
 	// internally.
 	case feePref.FeeRate != 0:
 		feePerKW := feePref.FeeRate
-		if feePerKW < chainfee.FeePerKwFloor {
-			log.Infof("Manual fee rate input of %d sat/kw is "+
-				"too low, using %d sat/kw instead", feePerKW,
-				chainfee.FeePerKwFloor)
 
+		// Because the user can specify 1 sat/vByte on the RPC
+		// interface, which corresponds to 250 sat/kw, we need to bump
+		// that to the minimum "safe" fee rate which is 253 sat/kw.
+		if feePerKW == chainfee.AbsoluteFeePerKwFloor {
```

Comment:
nit: would be nice to have the log showing the fee rate is replaced

Reply:
Aah right, added it back!